### DISCUSSION TITLE
Updating a mainnet verifier

### DISCUSSION THREAD

**Discussion by shayg-grappa** [Web3Auth/Web3Auth#1380] [Help]
We have an aggregate verifier set up on testnet and mainnet. Updating both has worked in the past several times. 
However, we are now not able to edit the mainnet verifier and we desperately need to.
Only option we get is `Show details` on the mainnet verifier whereas with the testnet we also get the `Update verifier` option

**Comment by ljx0520** [comment]
did you manage to do that?
